paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/FormArguments.kt

```diff
@@ -15,4 +17,6 @@ internal data class FormArguments(
     val shippingDetails: AddressDetails? = null,
     val billingDetailsCollectionConfiguration: PaymentSheet.BillingDetailsCollectionConfiguration =
         PaymentSheet.BillingDetailsCollectionConfiguration(),
+    val linkConfigurationCoordinator: LinkConfigurationCoordinator?,
+    val linkHandler: LinkHandler?,
```

Comment:
We might be able to inject these rather than make them form arguments if we make their creation in Dagger a `Singleton`.

Reply:
Could we get a reference to the view model instead or having to pass them through? What other alternatives? Is it bad to pass these in form arguments?